corehq/apps/hqwebapp/templates/hqwebapp/full_screen.html

```diff
@@ -3,18 +3,18 @@
 {% load menu_tags %}
 
 {% block content %}
-    {% block page_breadcrumbs %}{% endblock %}
-    <div class="container-fluid">
-        <div id="hq-content">
-            {% captureas page_title_block %}{% block page_title %}{% endblock page_title %}{% endcaptureas %}
-            {% if page_title_block %}
-            <div class="page-header" style="margin-top: 0;">
-                <h1 style="font-size: 1.8em;">{{ page_title_block }}</h1>
-            </div>
-            {% else %}
-            {% block pre_page_content %}<div class="spacer"></div>{% endblock %}
-            {% endif %}
-            {% block page_content %}{% endblock page_content %}
-        </div>
-    </div>
+  {% block page_breadcrumbs %}{% endblock %}
+  <div class="container-fluid">
+    <div id="hq-content">
+      {% captureas page_title_block %}{% block page_title %}{% endblock page_title %}{% endcaptureas %}
+      {% if page_title_block %}
+        <div class="page-header" style="margin-top: 0;">
+          <h1 style="font-size: 1.8em;">{{ page_title_block }}</h1>
+        </div>{# .page-header #}
```

Comment:
are these necessary / part of the new style? With everything properly indented, we shouldn't need these, especially not for things where the opening tag is one line away.

Reply:
I just like having these closing tags on divs as a best practice. I used this on dimagi.com @orangejenny should I keep these or nah?

Reply:
I like them for big blocks that don't fit on one screen but don't think they're necessary for small blocks (< 5-10 lines).

Reply:
This seems like the type of comment that can get stale and misleading very fast. Does your editor not tell you what the matching opening tag is?

Reply:
ah ok. should I leave the current comments in on this PR? moving forward I will only add the comment for longer blocks. how does this sound? @proteusvacuum @orangejenny

Reply:
Regarding staleness, I don't think we often change id/class names, at least not without grepping (since renaming has a good chance of causing js errors).

Searching briefly I found [this plugin](https://www.vim.org/scripts/script.php?script_id=3818) for vim but it highlights the matching tag, so I don't think it helps when the tag is off the screen. What does your editor do?

Reply:
If the matching tag is on screen it highlights it. If it is off screen, the matching tag shows up in the message bar at the bottom. 

![Screenshot from 2019-04-30 11-43-29](https://user-images.githubusercontent.com/146896/56974684-5123f980-6b3d-11e9-8382-14b38cf192e2.png)

![Screenshot from 2019-04-30 11-43-50](https://user-images.githubusercontent.com/146896/56974751-6d279b00-6b3d-11e9-87de-d683ae1ab93c.png)

Reply:
That is quite nice.

Reply:
Come join us at the Church of Emacs!

Reply:
Having the tag off screen might make having these comments even worse. If the dev can't see it when they update the class, then they won't be changing the comment.

Reply:
ok. i'll remove the comments